src/data/showcases.js

```diff
@@ -1127,6 +1127,15 @@ export const Showcases = [
     source: null,
     tags: ["lending", "token", "marketplace", "nftsupport"],
   },
+  {
+    title: "decon",
+    description:
+      "decon is a decentralized social forum.",
+    preview: require("./showcase/decon.png"),
+    website: "https://decon.app/",
+    source: null,
```

Comment:
```suggestion
    source: "https://github.com/alucao/decon",
```
I believe this will fix your build problem & would also enhance the appeal of listing. 🤓